core/api/request_api/RequestApi.py

```diff
@@ -11,19 +11,21 @@ def __init__(self):
         self.response = ""
         self.config = ConfigurationAuthentication()
 
-    def do_request(self, http_type, input_endpoint, map_object):
+    def do_request(self, http_type, input_endpoint, table_object, id_value):
         values = {}
-        for row in map_object:
+        for row in table_object:
             key_value = row['key']
-            value_value = row['value']
+            if key_value == "idBoard":
```

Comment:
why is this is hard-coded ?

Reply:
because I needed to comapare and asign a value to that paramenter, but I will improve it